theano/configparser.py

```diff
@@ -98,15 +98,17 @@ class change_flags(object):
 
     Useful during tests.
     """
-    def __init__(self, **kwargs):
+    def __init__(self, *args, **kwargs):
         confs = dict()
-        for k in kwargs:
+        args = dict(*args)
```

Comment:
I think you are miss reading. Here there isn't any constrain on how many args there is. The only constrain is that dict accept it, so that we can convert it to a dict.

The line 108 with the assert of len(l)==1, is there to make sure the key only correspond to 1 Theano flag. In fact, if this is false, the problem isn't with the parameter args, but with Theano itself. We should never have 2 Theano flags with the same name.